addons/point_of_sale/models/pos_category.py

```diff
@@ -22,6 +22,7 @@ def _check_category_recursion(self):
     child_id = fields.One2many('pos.category', 'parent_id', string='Children Categories')
     sequence = fields.Integer(help="Gives the sequence order when displaying a list of product categories.")
     image_128 = fields.Image("Image", max_width=128, max_height=128)
+    color = fields.Integer('Color', required=False, store=True, default=0)
```

Comment:
```suggestion
    color = fields.Integer('Color', required=False, default=0)
```
use store=True only for computed/related fields.

Reply:
:+1: